Bluepill-cli/BPInstanceTests/BPTestHelper.m

```diff
@@ -41,6 +41,11 @@ + (NSString *)sampleAppFatalTestsBundlePath {
 + (NSString *)resourceFolderPath {
     return [[NSBundle bundleForClass:[self class]] resourcePath];
 }
+
++ (NSString *)bpExecutablePath {
```

Comment:
The method bpExecutablePath is added to BPTesterHelper.m, which means it is intended to be used by test case only. We can find the bp executable from the bundle path In this case.

The code in BPRunner.m:BPRunnerForApp doesn't work when running as test cases, because the running process is not bluepill in this case, and the environment variables are not set yet.  It works when running standalone bluepill command in a released env.